demisto_sdk/commands/secrets/secrets.py

```diff
@@ -263,7 +263,7 @@ def get_related_yml_contents(self, file_path):
         # if script or readme file, search for yml in order to retrieve temp white list
         yml_file_contents = ''
         # Validate if it is integration documentation file or supported file extension
-        if checked_type(file_path, REQUIRED_YML_FILE_TYPES):
+        if find_type(file_path) in [FileType.INTEGRATION, FileType.SCRIPT]:
```

Comment:
also beta integration, no?

Reply:
it was a mistake
REQUIRED_YML_FILE_TYPES = [PACKS_INTEGRATION_PY_REGEX,
                           PACKS_INTEGRATION_README_REGEX,
                           PACKS_INTEGRATION_NON_SPLIT_README_REGEX,

                           PACKS_SCRIPT_PY_REGEX,
                           PACKS_SCRIPT_README_REGEX,
                           PACKS_SCRIPT_NON_SPLIT_README_REGEX,

                           PLAYBOOK_README_REGEX]
So only python and readme files.